day 4 homework

anton-vinogradov-su opened the pull request:
Я делал вот это ДЗ:
1) сделать класс Тестер наследуемым от класса TeamMember, удалить "лишний код" класса Тестер
2) Сделать класс ТестерСервис наследуемым от класса AbstractStorage, внести правки в логику класса ТестерСервайс исходя из нового наследования

Commit:
day 4 homework

aksodar commented on src/main/java/ru/sberbank/service/TesterService.java:
```diff
@@ -0,0 +1,22 @@
+public class TesterService extends AbstractStorage implements DataService<Tester> {
+    @Override
+    public boolean create(int id, String firstName, String secondName) {
+        return false;
```
почему пустой?

aksodar replied:
прицепи сюда, пожалуйста, старую реализацию, как мы это делали

aksodar commented:
не принято

Commit:
day 4 homework v2

Commit:
day 5 homework:
1 В TesterService унаследовать AbstractStorage и реализовать DataService с нужными дженериками и добавив логику методов (на лекции мы делали то же самое для DeveloperService).
2 Дополнительно в TesterService и DeveloperService реализовать метод get(Id) и его логику с учетом перегрузки)

anton-vinogradov-su commented:
ДЗ 5: 
1. В TesterService унаследовать AbstractStorage и реализовать DataService с нужными дженериками и добавив логику методов (на лекции мы делали то же самое для DeveloperService).
2. Дополнительно в TesterService и DeveloperService реализовать метод get(Id) и его логику с учетом перегрузки)

aksodar commented:
принято
